pages/Getting Started/Installation.md

```diff
@@ -38,7 +38,19 @@ hyprland - binary x86 tagged release
 ```
 
 {{< /tab >}}
-{{< tab "Nix" >}}Read the [Nix page](../../Nix).{{< /tab >}}
+
+{{< tab "Nix" >}}
+
+Enable Hyprland in your NixOS configuration:
+
+```nix
+programs.hyprland.enable = true;
+```
+
+For more details, read the [Nix page](../../Nix).
```

Comment:
Lets maybe mention that Hyprland is now in nixpkgs, and this option only enables the module from nixpkgs and not the flake, which would render a Hyprland input semi-useless. People have a tendency to copy others configurations and confuse themselves.

Reply:
That's what the "more details" is for.